test/fixtures/e2e/amp-video/videos-cdn.html

```diff
@@ -0,0 +1,137 @@
+      'i-amphtml-is-video-fully-loaded-override-for-testing';
+
+    window.onload = () => {
+      const debugField = document.querySelector('input#flexible-bitrate-debug');
+      debugField.addEventListener('keyup', (e) => {
+        const number = debugField.value.slice(0, -1);
+        const videoEl = document.querySelector(`amp-video#video${number} video`);
+        if (!videoEl) {
+          return;
+        }
+        if (e.key === 'd') {
+          videoEl.dispatchEvent(new Event('downgrade'));
+        } else if (e.key === 'l') {
+          videoEl.setAttribute(IS_VIDEO_FULLY_LOADED_OVERRIDE_FOR_TESTING, 'true');
+        } else if (e.key === 'u') {
+          videoEl.setAttribute(IS_VIDEO_FULLY_LOADED_OVERRIDE_FOR_TESTING, 'false');
+        } else {
+          return;
+        }
+        // If key was command, clean field.
+        debugField.value = '';
+      });
+      document.body.appendChild(debugField);
+    }
+    Object.defineProperty(window.navigator.connection, 'effectiveType', {
+      get: () => '4g'
+    });
+</script>
+</head>
+  <body>
+    <input id="flexible-bitrate-debug">
+    <amp-story standalone id="cats" supports-landscape
+        title="Key Highlights of AMP Conf 2018" publisher="The AMP team"
+        poster-portrait-src="logo.png"
+        publisher-logo-src="logo.png">
+
+      <amp-story-page id="page-1">
+        <amp-story-grid-layer template="fill">
+					<amp-video autoplay loop
+						id="video1"
+            width="400"
+            height="750"
+            poster="img-city1.jpeg#1"
+            layout="fill">
+            <source src="/examples/av/ForBiggerJoyrides-short.mp4#med1" type="video/mp4" data-bitrate="720">
+            <source src="/examples/av/ForBiggerJoyrides-short.mp4#low1" type="video/mp4" data-bitrate="400">
+            <source src="/examples/av/ForBiggerJoyrides-short.mp4#high1" type="video/mp4" data-bitrate="2000">
+          </amp-video>
+				</amp-story-grid-layer>
+      </amp-story-page>
+
+      <amp-story-page id="page-2">
+        <amp-story-grid-layer template="fill">
+					<amp-video autoplay loop
+					id="video2"
+            width="400"
+            height="750"
+            poster="img-city1.jpeg#2"
+            layout="fill">
+            <source src="/examples/av/ForBiggerJoyrides.mp4#med2" type="video/mp4" data-bitrate="720">
+            <source src="/examples/av/ForBiggerJoyrides.mp4#low2" type="video/mp4" data-bitrate="400">
+            <source src="/examples/av/ForBiggerJoyrides.mp4#high2" type="video/mp4" data-bitrate="2000">
+          </amp-video>
+				</amp-story-grid-layer>
+			</amp-story-page>
+
+			<amp-story-page id="page-3">
+        <amp-story-grid-layer template="fill">
+					<amp-video autoplay loop
+					id="video3"
+            width="400"
+            height="750"
+            poster="img-city1.jpeg#3"
+            layout="fill">
+            <source src="https://amp-dev.cdn.ampproject.org/c/s/amp.dev/static/samples/video/tokyo.mp4"
```

Comment:
Please add one test with this source as the `video[src]` and duplicate the extra test I asked for that checks the source generation with extra parameters